app/controllers/quizmaster/quizzes_controller.rb

```diff
@@ -79,4 +92,8 @@ def get_winner
     winner
   end
 
+  def quiz_params
+    params.permit(:name)
+  end
+
 end
```

Comment:
Ok while you're at it clean up the white space in this controller. (All the extra blank lines)